docs/guide/http_cache.md

```diff
@@ -88,127 +189,42 @@ services:
-
-``` php
-use Symfony\Component\HttpFoundation\Response;
- 
-// Inside a controller action
-// "Connects" the response to location #123 and sets a max age (TTL) of 1 hour.
-$response = new Response();
-$response->headers->set('X-Location-Id', 123);
-$response->setSharedMaxAge(3600);
-```
-
-#### Making your controller response context-aware
-
-If the content you're rendering depends on a user's permissions, then you should make the response context-aware:
-
-``` php
-use Symfony\Component\HttpFoundation\Response;
- 
-// Inside a controller action
-// Tells proxy configured to support this header to take the rights of a user (user hash) into account for the cache
-$response = new Response();
-$response->setVary('X-User-Hash');
-```
-
-## Configuration
-
-### Cache Purge
-
-This page explains the content cache purge *(aka invalidate)* mechanism used when publishing content from the UI or from a container-aware script, resulting in cache being invalidated either in the built-in Symfony Reverse Proxy, or on the much faster Varnish reverse proxy.
-
-Note that if you use a non-reverse proxy that does not support purge headers, shared content will stay in the cache for the whole duration defined by s-maxage, without the possibility of clearing it.
-
-#### Overview
-
-eZ Platform returns content-related responses with an `X-Location-Id` header that are stored together by the configured HTTP cache. This allows you to clear *(invalidate)* HTTP cache representing specifically a given Content item. On publishing the content, a cache purger is triggered with the Content ID in question, which in turn figures out affected content Locations based on [Smart HTTP cache clearing](#smart-http-cache-clearing) logic. The returned Location IDs are sent for purge using the purge type explained further below.
-
-#### Purge types
-
-##### Symfony Proxy: Local purge type
-
-By default, invalidation requests will be emulated and sent to the Symfony Proxy cache Store. Cache purge will thus be synchronous, meaning no HTTP purge requests will be sent around when publishing.
-
-``` yaml
-# ezplatform.yml
 ezpublish:
     http_cache:
         purge_type: local
 ```
 
 #### Varnish: HTTP purge type
 
-With Varnish you can configure one or several servers that should be purged over HTTP. This purge type is asynchronous, and flushed by the end of Symfony kernel-request/console cycle *(during terminate event)*. Settings for purge servers can be configured per site group or site access:
+With Varnish you can configure one or several servers that should be purged over HTTP.
+This purge type is asynchronous, and flushed by the end of Symfony kernel-request/console cycle (during the terminate event).
+Settings for purge servers can be configured per SiteAccess group or SiteAccess:
 
 ``` yaml
-# ezplatform.yml
```

Comment:
why file where this code can be found is removed? Maybe we can add info about it in the text as in case of: `app/config/security.yml` below?